src/Core/Startup/ParameterConfirmationToken.php

```diff
@@ -136,6 +168,16 @@ public function parameterProvided()
     }
 
     /**
+     * Is the parmeter requested in a BackURL param?
+     *
+     * @return bool
+     */
+    public function parameterProvidedBackURL()
```

Comment:
This naming is a bit weird, `hasParameterBackURL()` or `hasBackURL()` would be more suited?

Or if you'd like stick to "Provided" perhaps `parameterBackURLProvided()`?